examples/integration/test/fixtures/bwx.xcodeproj/project.pbxproj

```diff
@@ -8817,9 +8817,9 @@
 			files = (
 				A7EE330C67CB794EA5DC8FCE /* Assets.xcassets in Resources */,
 				96536BFC80D7064031732BC1 /* bucket in Resources */,
-				57B6669C9AE0F887A3D54BDA /* bucket in Resources */,
 				EB509AB2045BC989DBE6563C /* bucket in Resources */,
 				130B4EF21F478D0D76C741D0 /* bucket in Resources */,
+				57B6669C9AE0F887A3D54BDA /* bucket in Resources */,
```

Comment:
I'm hoping this is just a left over change that didn't get picked up somehow, and that the ordering is now stable...